### PR TITLE
Makes the game playable, add images and other minor changes

### PR DESCRIPTION
**Pull request by PlatinMTA**
This pull request will make the game playable without the need to refresh the page. It also deletes some duplicated code. See the commits for more information.

Request the changes you need and I'll gladly make those.

This also "fixes" #1 

We should talk about indentation and what method should we use. I'll make an issue about that.

### CODE REVIEW

File: blackjack.js [hunk 1]
```diff
@@ -47,26 +60,31 @@ function hitButton() {
...
   window.setTimeout(function () {
     dealCARDS(player);
   }, time);
 }
 
+// randomCard: Returns array with [0] being the card number and [1] being the suit
 function randomCard() {
-  let i = Math.floor(Math.random() * 13);
-  return blackjackGame["cards"][i];
+  let i = Math.floor(Math.random() * 13); // Maybe instead of declaring a variable move this piece of code directly into the array?
```
Comment (spoturno) [inline blackjack.js]: Good idea

File: blackjack.js [hunk 2]
```diff
@@ -89,66 +107,181 @@ function showScore(player) {
...
+	togButton("deal", false)
+
+	//make other buttons visible
+	togButton("hit", true)
+	togButton("stand", true)
+
+	//finish the game auto (TODO)
+}
+
+// togButton: string name, bool state [[, bool instant]]
```
Comment (spoturno) [inline blackjack.js:153]: Nice implementation ;)

**Review by spoturno — APPROVED** [review]
All approved

File: blackjack.js (hunk 1 above)
Comment (PlatinMTA) [reply blackjack.js]: I'll make the changes then

**Comment by PlatinMTA** [comment]
Thanks. Requested changes made. If it looks good to you then we should merge this.

**Pull request merged**